provider/oci/provider.go

```diff
@@ -206,34 +205,30 @@ func (e *EnvironProvider) Open(_ stdcontext.Context, params environs.OpenParams)
 	}
 
 	creds := params.Cloud.Credential.Attributes()
-	jujuConfig := common.JujuConfigProvider{
+	providerConfig := common.JujuConfigProvider{
 		Key:         []byte(creds["key"]),
 		Fingerprint: creds["fingerprint"],
 		Passphrase:  creds["pass-phrase"],
 		Tenancy:     creds["tenancy"],
 		User:        creds["user"],
-		OCIRegion:   creds["region"],
```

Comment:
I don't think this is correct, or at least it breaks existing deployments.

I think here we should check if the `creds["region"]` is empty, fallback to the `params.Cloud.Region`

Reply:
But this is the root of the problem...

If we do like this, we ignore the manually selected region by juju user:
```
juju bootstrap oracle/<selected-region-name>  oracle-ctrl
```
will take region defined in oracle creds.